index.js

```diff
@@ -1,15 +1,15 @@
 /**
  *  Next Snapchat Pixel Module
  *
- * @package next-snapchat-pixel
- * @author Kamal Mirzayev <mirzayevkamal@gmail.com>
+ * @package next-snapchat-pixel-fixed
+ * @author talha4real <talhabutt993@gmail.com>
```

Comment:
You don't change author or any details when you contribute to a repository. Please just commit if there is an issue and you have a fix for it